app/services/neighbour_response_creation_service.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class NeighbourResponseCreationService
+  class CreateError < StandardError; end
+
+  def initialize(planning_application:, **options)
+    @planning_application = planning_application
+    options.each { |k, v| instance_variable_set("@#{k}", v) unless v.nil? }
+  end
+
+  def call
+    save_neighbour_response!(build_neighbour_response)
+  end
+
+  private
+
+  attr_reader :local_authority, :params, :api_user
```

Comment:
Are you using `local_authority` or `api_user` ?
And could add planning_application to drop the @ below for consistency